scripts/kani-regression.sh

```diff
@@ -94,7 +94,24 @@ fi
 # Check codegen of firecracker
 time "$SCRIPT_DIR"/codegen-firecracker.sh
 
+# Test run 'cargo kani assess scan' on the existing workspace assess test
```

Comment:
Might be cleaner to move to a separate script that gets invoked by `kani-regression.sh`.

Reply:
Done.